File: Zestawy Zadań/C++/4 - Tablice/Zestaw 2 zadania/4.2.11 - Typ strukturalny typ1 z wartosciami z i y.cpp
#include <iostream>

using namespace std;

struct typ1
{
    int liczba;
    char znak;
};

int main()
{
    typ1 Z, Y;
    cout << "Podaj liczbe calkowita do Z: ";
    cin >> Z.liczba;
    cout << "Podaj znak do Z: ";
    cin >> Z.znak;
    cout << "Podaj liczbe calkowita do Y: ";
    cin >> Y.liczba;
    cout << "Podaj znak do Y: ";
    cin >> Y.znak;
    if (Z.liczba == Y.liczba && Z.znak == Y.znak)
    {
        cout << "Zgoda " << endl;
    }
    return 0;
}

File: Zestawy Zadań/Python/4 - Tablice/Zestaw 2 zadania/4.2.11 - Typ strukturalny typ1 z wartosciami z i y.py
class typ1:
    def __init__(self, liczba, znak):
        self.liczba = liczba
        self.znak = znak


z = typ1(0, "")
y = typ1(0, "")

z.liczba = int(input("Podaj liczbe calkowita do Z: "))
z.znak = str(input("Podaj znak do Z: "))
y.liczba = int(input("Podaj liczbe calkowita do Y: "))
y.znak = str(input("Podaj do Y: "))

if z.liczba == y.liczba and z.znak == y.znak:
    print("Zgoda")
